image/templates/helm/stackrox-central/templates/01-central-12-central-db.yaml

```diff
@@ -142,6 +146,10 @@ spec:
       volumes:
       - name: disk
         {{- toYaml ._rox.central.db.persistence._volumeCfg | nindent 8 }}
+      {{- if eq ._rox.env.installMethod "operator" }}
+      - name: backup
```

Comment:
I'm sorry to admit, but I forgot what is the intended behaviour in case the extension decides not to provision a backup PVC? 🤔 
Won't the deployment hang, waiting for a PVC to appear?

Reply:
Hm, yeah, the way it's implemented right now will lead to a pending PVC mount. Maybe it's worth to return `backup` variable back, and set it to false in this case. Would that work?

Reply:
I think it should. Perhaps the existence of a backup PVC created out-of-band should also influence the value of  `backup`?